src/backend/app.js

```diff
@@ -21,6 +22,78 @@ app.use(cors());
 
 router.use("/meals", mealsRouter);
 
+app.get("/future-meals", async (req, res) => {
+  try {  
```

Comment:
`let now = new Date()` shouldn't be able to fail, so it can be put outside the try block

Reply:
Thank you for all the feedback! I just made all the changes and made a new commit